content/workload-reduction-toolkit/evaluate-workload-measures/staff-workload-survey.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Staff workload survey
+layout: /markdown_resource.html.erb
+colour: green
+---
+
+# Evaluate impact using a staff workload survey
+
+<strong class="govuk-tag">Template</strong>
+
+Repeat the [survey](/workload-reduction-toolkit/identify-workload-issues/staff-workload-survey) in the
+Identify section of the toolkit regularly to gather feedback from school staff on the
+effectiveness of your workload reduction measures. You can use this feedback to evaluate and
+refine any actions you’ve taken to reduce workload.
+
+<div class="govuk-grid-row dfe-width-container govuk-!-padding-bottom-6">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/clipboard-icon.svg" alt="Clipboard icon">
+      </div>
+      <p>
+        The survey can be edited to meet the needs of your setting and
+        you can add, change or remove questions if required. For
+        example, some parts of the survey may be more appropriate for
+        primary or secondary settings.
+      </p>
+      <h2 class="govuk-heading-m">
+        Use online versions of the survey
+      </h2>
+      <p>
+        You can duplicate and adapt one of these online versions of the
+        staff workload survey:
+      </p>
+      <p>
+        <ul>
+          <li>
+            <a
+              href="https://forms.office.com/Pages/ShareFormPage.aspx?id=yXfS-grGoU2187O4s0qC-cn26r-uTMpNqURSfi9lRcVUNEg1UTdMMllFRTM1SEVRRDJWQjE3RUU5VS4u&sharetoken=MJnNysyL44umvL8f97JA"
+              class="govuk-link">
+              Microsoft Forms - staff workload survey template
+            </a>
+          </li>
+          <li>
```

Comment:
We are no longer using Google forms, please could we delete this?

I have to delete it from the identify one in another ticket :)